<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeTableWidgetUI</class>
 <widget class="QWidget" name="TimeTableWidgetUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>200</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="TimeTableWidgetGridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="removeRouteButton">
     <property name="toolTip">
      <string>Remove route</string>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="addRouteButton">
     <property name="toolTip">
      <string>Add new route</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="addRemoveButtonSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>221</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="4">
    <widget class="QTableWidget" name="routeListTable">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Train number</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Short description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Route</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="cloneRouteButton">
     <property name="toolTip">
      <string>Clone current route</string>
     </property>
     <property name="text">
      <string>Clone</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
